.github/workflows/ci.yaml

```diff
@@ -292,8 +292,6 @@ jobs:
             test_glob
             test_importlib
             test_io
-            test_iter
-            test_os
```

Comment:
This is a big progress without regression. just failing on CI.
Because it is not easily fixed now, let's revert this like to skip test_os on windows, and merge it first.
And lets think about this fix later.